data_hub/serializers.py

```diff
@@ -8,3 +8,15 @@ class FundamentalsAPIParamSerializer(serializers.Serializer):
     frequency = fields.ChoiceField(choices=["Quarterly", "Yearly"], required=True)
     year = fields.IntegerField(min_value=2016, required=False)
     quarter = fields.IntegerField(min_value=1, max_value=4, required=False)
+
+
+class MacroeconomicAPIParamSerializer(serializers.Serializer):
+    name = fields.CharField(required=True)
+    frequency = fields.ChoiceField(
+        choices=["Daily", "Weekly", "Monthly", "Quarterly", "Yearly"], required=False
+    )
+    from_date = fields.DateField(required=False)
+    to_date = fields.DateField(required=False)
+    year = fields.IntegerField(required=False)
+    month = fields.IntegerField(min_value=1, max_value=12, required=False)
+    quarter = fields.IntegerField(min_value=1, max_value=4, required=False)
```

Comment:
```suggestion
    quarter = fields.IntegerField(min_value=1, max_value=4, required=False)

```
nit: blank line at end of file